developerDocs/advanced-use-cases.md

```diff
@@ -0,0 +1,183 @@
+
+### Scheduling Future Listings
+
+You can create sell orders that aren't fulfillable until a future date. Just pass in a `listingTime` (a UTC timestamp in seconds) to your SDK instance:
+
+```typescript
+const auction = await openseaSDK.createSellOrder({
+  tokenAddress,
+  tokenId,
+  accountAddress,
+  startAmount: 1,
+  listingTime: Math.round(Date.now() / 1000 + 60 * 60 * 24), // One day from now
+});
+```
+
+### Purchasing Items for Other Users
+
+You can buy and transfer an item to someone else in one step! Just pass the `recipientAddress` parameter:
+
+```typescript
+const order = await openseaSDK.api.getOrder({ side: "ask", ... })
+await openseaSDK.fulfillOrder({
+  order,
+  accountAddress, // The address of your wallet, which will sign the transaction
+  recipientAddress // The address of the recipient, i.e. the wallet you're purchasing on behalf of
+})
+```
+
+If the order is a sell order (`order.side === "ask"`), the taker is the _buyer_ and this will prompt the buyer to pay for the item(s) but send them to the `recipientAddress`. If the order is a buy order ( `"bid"`), the taker is the _seller_ but the bid amount be sent to the `recipientAddress`.
+
+This will automatically approve the assets for trading and confirm the transaction for sending them.
+
+### Using ERC-20 Tokens Instead of Ether
+
+Here's an example of listing the Genesis CryptoKitty for $100! No more needing to worry about the exchange rate:
+
+```typescript
+// Token address for the DAI stablecoin, which is pegged to $1 USD
+const paymentTokenAddress = "0x89d24a6b4ccb1b6faa2625fe562bdd9a23260359";
+
+// The units for `startAmount` and `endAmount` are now in DAI, so $100 USD
+const auction = await openseaSDK.createSellOrder({
+  tokenAddress: "0x06012c8cf97bead5deae237070f9587f8e7a266d", // CryptoKitties
+  tokenId: "1", // Token ID
+  accountAddress: OWNERS_WALLET_ADDRESS,
+  startAmount: 100,
+  paymentTokenAddress,
+});
+```
+
+You can use `getPaymentTokens` to search for tokens by symbol name. And you can even list all orders for a specific ERC-20 token by querying the API:
+
+```typescript
+const token = (await openseaSDK.api.getPaymentTokens({ symbol: "MANA" }))
+  .tokens[0];
+
+const order = await openseaSDK.api.getOrders({
+  side: "ask",
+  paymentTokenAddress: token.address,
+});
+```
+
+**Fun note:** soon, all ERC-20 tokens will be allowed! This will mean you can create crazy offers on crypto collectibles **using your own ERC-20 token**. However, opensea.io will only display offers and auctions in ERC-20 tokens that it knows about, optimizing the user experience of order takers. Orders made with the following tokens will be shown on OpenSea:
+
+- MANA, Decentraland's currency: https://etherscan.io/token/0x0f5d2fb29fb7d3cfee444a200298f468908cc942
+- DAI, Maker's stablecoin, pegged to $1 USD: https://etherscan.io/token/0x89d24a6b4ccb1b6faa2625fe562bdd9a23260359
```

Comment:
this is the old SAI, should be updated to DAI contract https://etherscan.io/token/0x6b175474e89094c44da98b954eedeac495271d0f